requirements.txt

```diff
@@ -0,0 +1,2 @@
+pytest
+git+https://github.com/f02h/lighthouse-python.git#egg=lighthouse
```

Comment:
I know we discussed this in person, but for visibility I thought I'd comment the outcome of that. 

At the moment we aren't relying on our own fork because the current implementation doesn't rely on any changes to the package code.
When we come to add lighthouse integration to more projects, it is extremely likely that we will being developing our fork to have additional functionality and at that point we can make the switch to our own version (and hopefully as a PyPi package).